### DISCUSSION TITLE
Codespace stuck in "starting loop", crashes browser

### DISCUSSION THREAD

**Discussion by bradtaniguchi** [community/community#8160] [Codespaces] [answered]
When opening one of my codespaces I see "starting your codespace" message, and the page quickly reloads, and starts redirecting between the following URLS:
- `https://bradtaniguchi-<codespace-id>.github.dev`
- `https://github.com/codespaces/bradtaniguchi-<codespace-id>`

More recently I'll see the redirects for only a few seconds before chrome crashes. 

**system info**
OS: ChromeOS
```
Version 97.0.4692.27 (Official Build) beta (64-bit)
```

---

I have them set up as PWA's, and have only seen this issue today but haven't had issues in the past.

- Deleting and re-creating the codespaces also had no effect.
- Removing the PWA had no effect.

**Comment by WikiRik** [comment]
I have the same issue, but the same Codespace works fine on Chrome 96 (aka the latest non-beta version). So it might be an issue with Chrome instead of Codespaces

It's also not for all Codespaces

**Reply by bradtaniguchi** [reply]
Glad to hear I'm not the only one out there seeing issues! :D


I'll install chrome 97 on a windows machine, and re-check with chrome 96 stable on the same machine to see if I can replicate it. If so, then yea it must be isolated to the 97 browser/beta version.

**Comment by Hazmi35** [comment, marked as answer]
Clearing cookies on this domain works though.
![image](https://user-images.githubusercontent.com/32807631/144752254-612e3e63-41f5-414c-b57f-850a94bbb7ed.png)

The settings are in `chrome://settings/siteData?searchSubpage=github.dev`. Only clear the `<username>-<codespace id>.github.dev`, but it will probably clear user data like unsaved files, manually installed extensions, manually set config, etc. but I don't do that because I let my `.devcontainer` folder and vscode sync to do those

**Comment by JavaDerg** [comment]
I'm on firefox 100.0.2 and it does the same
Clearing the cookies does not work.
Nor does it work in a private window.
